### DISCUSSION TITLE
Unable to host site build using vite on GitHub Pages

### DISCUSSION THREAD

**Discussion by rudrakshkarpe** [antfu/vitesse#394] [Q&A]
### Describe the bug

-  blank white page error appeared tried to follow Github documentation and hosting guidelines but it's the same
-  I am using ```.me``` custom domain from **Namecheap** please help me fix the issue 
- I am excited to host my portfolio with this stack

### Reproduction

Unable to host site build using vite on GitHub Pages

### System Info

```Shell
-  blank white page error appeared tried to follow Github documentation and hosting guidelines but it's the same
-  I am using .me``` custom domain from **Namecheap** please help me fix the issue 
- I am excited to host my portfolio with this stack
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by lixianbin1** [comment]
You can try adding a blank file `.nojekyll`
